packages/core/admin/admin/src/pages/Layout.tsx

```diff
@@ -157,10 +161,12 @@ const AdminLayout = () => {
                 {formatMessage({ id: 'skipToContent', defaultMessage: 'Skip to content' })}
               </SkipToContent>
               <Flex alignItems="flex-start">
-                <LeftMenu
-                  generalSectionLinks={generalSectionLinks}
-                  pluginsSectionLinks={pluginsSectionLinks}
-                />
+                {!isHistoryRoute && (
```

Comment:
do you still need this considering your changes in `App`?

Reply:
yes because there are two different things called left menu:
- in App, it's the app-wide sidebar
- here, it's the list of content types

we want neither for history

Reply:
Ah sorry, i misread the path and thought this was the layout for the CM! Did you consider using a portal to the document.body and using css `https://developer.mozilla.org/en-US/docs/Web/CSS/isolation` to avoid needing to add these checks?

What's your plan for hiding the side bar when the CM moves out of the admin package?

I'm basically wondering if your solution could be wider used by others in the future for other use cases 🤔

Reply:
I hadn't thought about it but just tried it. By default, it displays the page after a full viewport scroll. I have to use position fixed with z-index and a background color for it to work.

So it does work, but it feels hacky to me. I'm afraid having a full scrollable page overlayed on other content would create accessibility problems. For example, I don't get the native Chrome spring/bouce animation when trying to scroll past the content.

So I'm not a big fan, but what do you think?

https://github.com/strapi/strapi/assets/8087692/a6803b64-97ab-4e08-aadb-b809549ec4f5

Reply:
>What's your plan for hiding the side bar when the CM moves out of the admin package?

I was thinking of exporting the useIsHistoryRoute from the content manager package. Worst case scenario we could also hardcode the history routes in the admin. And I don't think the we'll have the need for another full screen page, this one is an exception imo

Reply:
I don't think we'll need to block more stuff, things like the help button or the guided tour are fine to keep

I think I prefer to the current implementation because it's simpler, at least for now. We can see when we migrate the CM if it makes more sense to change it